src/fswAlgorithms/pointCloudProcessing/pointCloudTriangulation/pointCloudTriangulation.cpp

```diff
@@ -83,12 +82,11 @@ void PointCloudTriangulation::UpdateState(uint64_t currentSimNanos)
             this->measuredPointCloud.emplace_back(featureLocation);
         }
         this->pointCloudSize = this->numberKeyPoints;
+        this->numberTimesCalled += 1;
```

Comment:
:large_orange_diamond: Should we update the member variable's comment in the header to say e.g. "Number of times called with valid input data..."

Reply:
Good idea!